packages/vaadin-ordered-layout/imports.d.ts

```diff
@@ -1 +1,3 @@
+export * from './src/vaadin-horizontal-layout.js';
+export * from './src/vaadin-vertical-layout.js';
```

Comment:
This two were missing I guess? They are exported from the corresponding JS file.

Reply:
Good catch, thanks.